src/fontra/views/editor/scene-controller.js

```diff
@@ -504,6 +504,41 @@ export class SceneController {
     return undoInfo !== undefined;
   }
 
+  async doDelete() {
+    await this.editInstance((sendIncrementalChange, instance) => {
+      const { point: pointSelection } = parseSelection(this.selection);
+
+      const changes = recordChanges(instance, (instance) => {
+        const path = instance.path;
+
+        let offset = 0;
+        for (const point of pointSelection) {
```

Comment:
Please rename `point` to `pointIndex`

Reply:
Also, if you change it to 
```js
for (const pointIndex of reversed(pointSelection)) {
```
You can get rid of the `offset` logic.

(`reversed` lives in utils.js)

(The output of `parseSelection` is guaranteed to be sorted)